schemas/xml/AAS.xsd

```diff
@@ -58,7 +58,7 @@
 			<group ref="aas:hasDataSpecification"/>
 			<element maxOccurs="1" minOccurs="0" name="security" type="abac:security_t"/>
 			<element maxOccurs="1" minOccurs="0" name="derivedFrom" type="aas:reference_t"/>
-			<element maxOccurs="unbounded" minOccurs="0" name="submodelRefs" type="aas:submodelRefs_t"/>
+			<element maxOccurs="unbounded" minOccurs="0" name="submodel" type="aas:submodel_t"/>

```

Comment:
submodelRefs_t is correct, it is a shared aggregation and shared aggregation are always serialized as reference. But perhaps maxOccurs needs to be "1"